classification/classification_network.py

```diff
@@ -241,108 +230,132 @@ def createClassifierRegion(network, classifierType, classifierParams, prevRegion
     # Add new region class to the network
     network.registerRegion(SequenceClassifierRegion)
     PY_REGIONS.append(classifierType.split(".")[1])
-  
+
   # Create the classifier region.
   classifierRegion = network.addRegion(
-      "classifier", classifierType, json.dumps(classifierParams))
+    "classifier", classifierType, json.dumps(classifierParams))
 
   # Disable learning for now (will be enabled in a later training phase)... why???
   classifierRegion.setParameter("learningMode", False)
-  
+
   # Okay to always leave inference mode on; only there for some corner cases.
   classifierRegion.setParameter("inferenceMode", True)
 
+  if classifyPredictedActiveCells:
```

Comment:
i removed this notion of classifyPredictedActiveCells for now. i am going to tweak the tm states classification differently, after having a discussion with subutai